segmentation_models_pytorch/encoders/__init__.py

```diff
@@ -49,17 +49,25 @@
 
 
 def get_encoder(name, in_channels=3, depth=5, weights=None, output_stride=32, **kwargs):
-
     if name.startswith("tu-"):
         name = name[3:]
-        encoder = TimmUniversalEncoder(
-            name=name,
-            in_channels=in_channels,
-            depth=depth,
-            output_stride=output_stride,
-            pretrained=weights is not None,
-            **kwargs,
-        )
+        if name.startswith("vit"):
```

Comment:
could you please go through other archs except vit in timm and check whether they are supported?
if some other archs are supported we probably should change the logic here